guide/interactions/context-menus.md

```diff
@@ -3,7 +3,7 @@
 Context Menus are application commands which appear when right clicking or tapping a user or a message, in the Apps submenu.
 
 ::: tip
-This page is a follow-up to the [interactions (slash commands) page](/interactions/slash-commands.md). Please carefully read that section first, so that you can understand the methods used in this section.
+This page is a follow-up to the [slash commands page](/slash-commands/advanced-creation.md). Please carefully read those first so that you can understand the methods used in this section.
```

Comment:
plural "those" for singular in prev. sentence, if more than a page, rephrase prev.